lib/fathead/javascript_eslint/doc/notes.md

```diff
@@ -0,0 +1,52 @@
+# Notes
```

Comment:
I would just move the important parts of this into the README, and make that a markdown file.

Any reason you've created /doc/notes.md?